### PR TITLE
3 more gfx functions

### PR DESCRIPTION
**Pull request by Unnunu**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀 This PR matches 3 functions (+0.22%) on `us`.

File: src/C50A0.c [hunk 1]
```diff
@@ -2222,19 +2799,19 @@ void func_801363A0(ItemEntity* itemEntity) {
         case 10:
             if (!(itemData->typeFlags & ITEM_TYPE_FLAG_BADGE)) {
                 if (!(itemEntity->flags & ITEM_ENTITY_FLAGS_4000000) || (itemEntity->pickupMsgFlags & 0x4)) {
-                    itemMsg = MSG_Menus_0058;
+                    itemMsg = MESSAGE_ID(0x1D, 0x058);
```
Comment (ethteck) [inline src/C50A0.c:2802]: Is there a reason you went back to the more primitive macros? Do the MSG_ defines not work?

**Review by ethteck — APPROVED** [review]

File: src/C50A0.c (hunk 1 above)
Comment (Unnunu) [reply src/C50A0.c:2802]: I didn't decompile this function,I just copied old code from decomp.me and made some score improvement. Is it ok to leave it as is?

**Comment by ethteck** [comment]
Yesh

File: src/C50A0.c (hunk 1 above)
Comment (HailSanta) [reply src/C50A0.c:2802]: Would be much better to keep the lines with MSG_* and WINDOW_ID_*.

Comment (Unnunu) [reply src/C50A0.c]: done

**Review by nanaian — APPROVED** [review]

**Pull request merged**